From 10b245589fa04aa5b1893dbc1694118ab242db96 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Wed, 4 Oct 2023 17:28:20 +0200
Subject: [PATCH] drm: bridge: adv7511: fill i2s stream capabilities

Set no_i2s_capture flag in hdmi_codec_pdata structure to report
that the ADV7535 HDMI bridge does not support i2s audio capture.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: Ie56518951348e20fdc6f945ec5943761a84e134b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/376689
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 drivers/gpu/drm/bridge/adv7511/adv7511_audio.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/bridge/adv7511/adv7511_audio.c b/drivers/gpu/drm/bridge/adv7511/adv7511_audio.c
index 98f1a9f34c18..ce6eda87bfe0 100644
--- a/drivers/gpu/drm/bridge/adv7511/adv7511_audio.c
+++ b/drivers/gpu/drm/bridge/adv7511/adv7511_audio.c
@@ -237,6 +237,7 @@ static const struct hdmi_codec_pdata codec_data = {
 			SNDRV_PCM_FMTBIT_IEC958_SUBFRAME_LE),
 	.max_i2s_channels = 2,
 	.i2s = 1,
+	.no_i2s_capture = 1,
 	.spdif = 1,
 };
 
-- 
2.39.5

